packages/core/src/wallet/metamaskBridge.ts

```diff
@@ -0,0 +1,34 @@
+import type { MetaMaskProvider } from '@consensys/get-starknet/dist/type';
```

Comment:
`import type { MetaMaskProvider } from '@consensys/get-starknet';` is available in the new package

Reply:
resolved